sql/logic_test.go

```diff
@@ -376,10 +376,14 @@ func (t *logicTest) execStatement(stmt logicStatement) {
 	switch {
 	case stmt.expectErr == "":
 		if err != nil {
-			t.Fatalf("%s: expected success, but found %v", stmt.pos, err)
+			t.Fatalf("%s: expected success, but found %q", stmt.pos, err)
```

Comment:
I changed some of them earlier to make be able to tell when the problem was due to escaping. much more convenient that way.